src/main/java/hudson/tasks/junit/History.java

```diff
@@ -483,24 +421,16 @@ private ObjectNode computeResultTrendJson(List<HistoryTestResultSummary> history
     private ObjectNode computeDistributionJson(List<HistoryTestResultSummary> history) {
         ObjectNode root = MAPPER.createObjectNode();
         ArrayNode series = MAPPER.createArrayNode();
-        ArrayNode domainAxisLabels = MAPPER.createArrayNode();
 
         ObjectNode durationSeries = MAPPER.createObjectNode();
         durationSeries.put("name", "Build Count");
-        durationSeries.put("type", "line");
-        durationSeries.put("symbol", "circle");
-        durationSeries.put("symbolSize", "0");
-        durationSeries.put("sampling", "lttb");
+        durationSeries.put("type", "bar");
+        durationSeries.put("barWidth", "99%");
         ArrayNode durationData = MAPPER.createArrayNode();
         durationSeries.set("data", durationData);
         ObjectNode durationStyle = MAPPER.createObjectNode();
         durationSeries.set("itemStyle", durationStyle);
         durationStyle.put("color", "--success-color"); // "rgba(50, 200, 50, 0.8)");
```

Comment:
This color is a bit ugly now that it is not just a simple line in case anyone has any recommendation. Ideally this would be a stacked histogram chart with each build result shown in a different color.